packages/p2p/src/components/payment-method-card/payment-method-card.tsx

```diff
@@ -133,7 +133,11 @@ const PaymentMethodCard = ({
                         suffix_icon='IcCashierVerticalEllipsis'
                     />
                 )}
-                {(active_index === 2 || active_index === 0) && (
+                {/* Display the checkbox selection only when payment method card is shown from
+                buy/sell tab(active_index === 0) or from my-ads tab(active_index === 2) or when its advertiser page */}
+                {(active_index === 2 ||
+                    active_index === 0 ||
+                    (active_index === 3 && buy_sell_store.show_advertiser_page)) && (
```

Comment:
since we are not calling this component in orders page, can you please check if we can remove this condition instead?